signbank/dictionary/tests/test_update.py

```diff
@@ -358,7 +358,7 @@ def test_confirmation_view_confirm_gloss_creation(self):
         gloss = gloss_qs.first()
         self.assertEqual(f"{csv_content['word']}:{gloss.pk}", gloss.idgloss)
         self.assertEqual(f"{maori_words[0]}:{gloss.pk}", gloss.idgloss_mi)
-        self.assertEqual(csv_content["notes"], gloss.notes)
+        # self.assertEqual("", gloss.notes)
```

Comment:
```suggestion
        self.assertEqual("", gloss.notes)
```
**suggestion**: if this should be blank let's assert that, otherwise maybe we should just delete the line?